Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -859,6 +859,16 @@ final class LayerTests: XCTestCase {
         let layer = GlobalMinPool1D<Float>()
         let input = Tensor(shape: [2, 2, 2], scalars: (0..<8).map(Float.init))
         let computedGradient = gradient(at: input, layer) { $1($0).sum() }
+        // The expected value of the gradient was computed using the following Python code:
+        // ```
+        // import tensorflow as tf
+        // from tensorflow.python.keras import backend
+        // x = tf.reshape(tf.range(8, dtype=tf.float32), [2, 2, 2])
+        // with tf.GradientTape() as tape:
+        // tape.watch(x)
```

Comment:
I don't think this is aligned properly.

Reply:
Yea, I noticed it right after pushing. Fixed.